src/legacy/core_plugins/kibana/public/discover/np_ready/angular/helpers/_get_point.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Aspect, Table, Row } from './point_series';
+
+type RowValue = number | 'NaN';
+interface Raw {
+  table: Table;
+  column: number;
+  row: number;
+  value: RowValue;
+}
+export interface Point {
+  x: RowValue;
+  y: RowValue;
+  extraMetrics: [];
+  xRaw: Raw;
+  yRaw: Raw;
+  tableRaw?: {
+    table: Table;
+    column: number;
+    row: number;
+    value: number;
+    title: string;
+  };
+  parent: null;
+  series?: string;
+  seriesId?: string;
+}
+
+export function getPoint(table: Table, x: Aspect, row: Row, rowIndex: number, y: Aspect) {
```

Comment:
It seems like the `Aspect` concept is overkill for the discover case - we can probably just hardcode the `x` and `y` accessors in this function (and get rid of the raw stuff)